File: WebApi/Angular/src/app/_models/Dishes/Dish.ts
import { Ingredient } from "../Ingredients/Ingredient";

export interface Dish {
    id: number;
    dishName: string;
    dishPrice: number;
    dishDescription: string;
    dishStatusId: number;
    dishStatusName: string; 
    dishCategoryId: number;
    dishCategoryName: string;
    ingredientsId: number[];
  }
  

File: WebApi/Domain/Entities/Dish.cs
using System.Collections.Generic;

namespace Domain.Entities
{
    public class Dish : BaseEntity
    {
        public string DishName { get; set; }
        public int DishPrice { get; set; }
        public string DishDescription { get; set; }
        public int DishCategoryId { get; set; }
        public DishCategory DishCategory { get; set; }
        public int DishStatusId { get; set; }
        public DishStatus DishStatus { get; set; }
        public ICollection<DishIngredient> DishIngredients { get; set; }
        public ICollection<DishOrder> DishOrders { get; set; }
    }
}